proc_CPMG_mw.py

```diff
@@ -116,16 +115,9 @@ def print_fun(x, f, accepted):
                 ydata *= -1
             ydata /= max(ydata)
             fl.plot(x,ydata, '.', alpha=0.4, label='data', human_units=False)
-            fitfunc = lambda p, x: p[0]*exp(-x*p[1])
-            errfunc = lambda p_arg, x_arg, y_arg: fitfunc(p_arg, x_arg) - y_arg
-            p0 = [0.1,100.0]
-            p1, success = leastsq(errfunc, p0[:], args=(x, ydata))
-            assert success == 1, "Fit did not succeed"
-            T2 = 1./p1[1]
-            x_fit = linspace(x.min(),x.max(),5000)
-            fl.plot(x_fit, fitfunc(p1, x_fit),':', label='fit (T2 = %0.2f ms)'%(T2*1e3), human_units=False)
-            xlabel('t (sec)')
-            ylabel('Intensity')
+            raise RuntimeEror("Need to set up fitdata using new capabilities")
+            data.guess([0.1,100.0]) <-- should be a dictionary
+            fl.plot(data.eval(400))
```

Comment:
I'm thinking just leave this for a future pull request (leave non-functional for now) -- too much in this one

Reply:
actually, I created an issue so we can leave this alone for now